### ISSUE TITLE
Missing Documentation For Hosting Flutter App With Serverpod On GCP

### ISSUE THREAD

**Issue by Kiwi-KiwiCorp** [serverpod/serverpod#1959]
**Is your feature request related to a problem? Please describe.**
Currently, the only documentation on how to host your application using serverpod is the Pixorama example however that only contains a recommendations on how to update the deploy-aws file.

**Describe the solution you'd like**
A devoted page in the documentation for hosting a web app with serverpod with both AWS and GCP variations would be much appreciated.

**Comment by EquosNine** [comment]
There is this guide: https://docs.serverpod.dev/deployments/deploying-to-gce-terraform
but I was not able to get it working the last time I tried to deploy serverpod to a GCP instance.

Your Flutter Application, from my understanding will need to be hosted separately based on the platform you are developing for.

ie Firestore Hosting for the Web application or an Android app on the play store. Atleast that's the route I would take.

**Comment by kieran-novarca** [comment]
> There is this guide: https://docs.serverpod.dev/deployments/deploying-to-gce-terraform but I was not able to get it working the last time I tried to deploy serverpod to a GCP instance.
> 
> Your Flutter Application, from my understanding will need to be hosted separately based on the platform you are developing for.
> 
> ie Firestore Hosting for the Web application or an Android app on the play store. Atleast that's the route I would take.

No you can actually host your web application using serverpod itself. The pixorama example is doing this and actually has details on how you can do this yourself at the bottom of the github readme. This issue was raised because
1. Not a lot of people know about this as there is no official documentation for it (just the example) 
2. The example uses AWS and not google for the deployment

I have actually got this working and have currently got a test app that is currently deployed and hosted using GCP, it just took me 3 days of trial and error to get it working, so documentation would be quite nice

**Comment by jblankenship5** [comment]
Agreed, I would love some documentation on how to get my Flutter web app up and running on GCP

**Comment by iampopal** [comment]
We also need to upload the web app into GCP Cloud Run. If someone please help will be awesome.
